dice/apps/rounds/views.py

```diff
@@ -16,15 +16,15 @@ class RoundViewSet(viewsets.mixins.CreateModelMixin, viewsets.mixins.RetrieveMod
 
     def extra_validation(self, game):
         if Round.objects.filter(user=self.request.user, figure__isnull=True, game=game).exists():
-            raise PermissionDenied('Istenieje niezakończona runda, nie można stworzyć kolejnej')
+            raise RoundExistsError
```

Comment:
Let's raise exceptions instances not exceptions classes:

```suggestion
            raise RoundExistsError()
```